README.md

```diff
@@ -6,6 +6,48 @@ A simple wrapper to interact with RAG
 
 Contributions are welcome. Take a look at [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to get started.
 
+## Trying CLA
+
+The `Command Line Assistant` client RPM is available for some versions of RHEL, currently it
+is available for RHEL-10.
```

Comment:
```suggestion
is available for RHEL 10 and RHEL 9.
```

We also ship for rhel 9

Reply:
Thank you for the feedback @r0x0d 
My commit [97f3285](https://github.com/rhel-lightspeed/command-line-assistant/pull/362/commits/97f328518b2c5d471c77e99c02c7c8086f1e33cc) should address this.